src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CashoutResult.cs

```diff
@@ -4,6 +4,7 @@ public enum CashoutResult
     {
         Unknown,
         Success,
-        Failure
+        Failure,
+        AmountTooSmall
```

Comment:
I believe value AmountTooSmall not needed for CashoutResult

Reply:
I suggest to add ErrorCode nullable enum to Aggregate. Values - nearly the same as ErrorCode in CashoutFailedEvent